src/components/Layout/Auth/EasyLogin.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import kakaotalkLoge from "../../../assets/images/social/kakaotalk-logo.svg";
+import googleLoge from "../../../assets/images/social/google-logo.svg";
+
+interface Image {
+  src: string;
+  alt: string;
+  width: number;
+}
+
+interface Link {
+  href: string;
+  target: string;
+  rel: string;
+  img: Image;
+}
+
+const Links: { [id: string]: Link } = {
+  googleLink: {
+    href: "https://www.google.com/",
+    target: "_blank",
+    rel: "noopener noreferrer",
+    img: {
+      src: googleLoge,
+      alt: "구글",
+      width: 42,
+    },
+  },
+  kakaoLink: {
+    href: "https://www.kakaocorp.com/page/",
+    target: "_blank",
+    rel: "noopener noreferrer",
+    img: {
+      src: kakaotalkLoge,
+      alt: "카카오톡",
+      width: 42,
+    },
+  },
+};
+
+const EasyLogin: React.FC = () => {
+  return (
+    <section className="easy-login">
+      <h3>간편 로그인하기</h3>
+      <div>
+        {Object.entries(Links).map(([key, { href, target, rel, img }]) => (
+          <a key={key} href={href} target={target} rel={rel}>
+            <img src={img.src} alt={img.alt} width={img.width} />
```

Comment:
만약, 위와 같이 속성의 이름이 완전히 일치하는 상황이 아니라면 이중으로 구조분해 할당을 사용하는 간접적인 방식도 있습니다. (다만, 이 경우에는 굳이 이렇게 길게 작성하기 보다는 위 쪽 코멘트처럼 작성하는 것이 좋아보이네요.)

```suggestion
        {Object.entries(Links).map(([key, { href, target, rel, img: { src, alt, width } }]) => (
          <a key={key} href={href} target={target} rel={rel}>
            <img src={src} alt={alt} width={width} />
          </a>
  ))}
```

Reply:
넵 이중구조분해 할당으로 하는 게 더 깔끔하네요~!! 이 정도 길이는 괜찮을 거 같아서 바꿨습니다.😆